build/build-proto.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -o errexit
+set -o xtrace
+
+if ! swagger version &>/dev/null; then
+  go install github.com/go-swagger/go-swagger/cmd/swagger
```

Comment:
use tag for repeatable build
```suggestion
  go get github.com/go-swagger/go-swagger/cmd/swagger@v0.24.0
```